bin/p-s.js

```diff
@@ -46,7 +47,7 @@ runPackageScript({
 }, function(result) {
   if (result.error) {
     log.error(result.error)
-    const FAIL_CODE = 1
```

Comment:
Yikes! Thanks for noticing that! Would you like to [makeapullrequest.com](http://makeapullrequest.com)?